tensorflow/c/c_api_experimental.h

```diff
@@ -305,7 +305,9 @@ TF_ImportGraphDefOptionsSetValidateColocationConstraints(
     TF_ImportGraphDefOptions* opts, unsigned char enable);
 
 // Load the library specified by library_filename and register the pluggable
-// device and related kernels present in that library.
+// device and related kernels present in that library. This function should work
```

Comment:
Please rephrase to:

"This function is not supported on embedded on mobile and embedded platforms and will fail if called."

Reply:
done